queries/purescript/highlights.scm

```diff
@@ -27,6 +27,8 @@
  [
  (comma)
  ";"
+ (qualified_module)  ; grabs the `.` (dot), ex: import System.IO
+ "."
```

Comment:
Do you need both?

Reply:
Yes, dots are also used for record access, not just modules.